actors/miner/src/policy.rs

```diff
@@ -22,15 +20,18 @@ use super::{PowerPair, BASE_REWARD_FOR_DISPUTED_WINDOW_POST};
 /// Precision used for making QA power calculations
 pub const SECTOR_QUALITY_PRECISION: i64 = 20;
 
+/// Unit of sector quality
+pub type SectorQuality = BigInt;
+
 lazy_static! {
     /// Quality multiplier for committed capacity (no deals) in a sector
-    pub static ref QUALITY_BASE_MULTIPLIER: BigInt = BigInt::from(10);
+    pub static ref QUALITY_BASE_MULTIPLIER: SectorQuality = BigInt::from(10);
 
     /// Quality multiplier for unverified deals in a sector
-    pub static ref DEAL_WEIGHT_MULTIPLIER: BigInt = BigInt::from(10);
+    pub static ref DEAL_WEIGHT_MULTIPLIER: SectorQuality = BigInt::from(10);
 
     /// Quality multiplier for verified deals in a sector
-    pub static ref VERIFIED_DEAL_WEIGHT_MULTIPLIER: BigInt = BigInt::from(100);
+    pub static ref VERIFIED_DEAL_WEIGHT_MULTIPLIER: SectorQuality = BigInt::from(100);
```

Comment:
Is this file the only use of SectorQuality? If so I think it's only adding noise here and we should just drop the type alias.